src/Transport/NServiceBus.Transport.AzureServiceBus.csproj

```diff
@@ -12,6 +12,8 @@
     <PackageReference Include="NServiceBus" Version="[7.0.1, 8.0.0)" />
     <PackageReference Include="Particular.CodeRules" Version="0.2.1" PrivateAssets="All" />
     <PackageReference Include="Particular.Packaging" Version="0.6.2" PrivateAssets="All" />
+    <PackageReference Include="Fody" Version="6.3.0" PrivateAssets="All" />
```

Comment:
I'd prefer to see these alphabetized.

Reply:
I'll push back on this :)

Reply:
Why? If they are in that ItemGroup, they should be alphabetized. I'd be fine with them being a in a separate group if you really want to separate them in some way, but as long as they stay in that one, sort them.

Reply:
I like to keep related packages together. With Fody going on top and obsolete.fody to the bottom, the sorting will be alphabetical but the connection is not obvious.